### PR TITLE
Add token auth for etl->api communication

### PR DESCRIPTION
**Pull request by nikitazigman**
* Add a token-based auth 
* Add permissions checkers for bulk views 
* Add a token header to etl

### CODE REVIEW

File: backend/apps/beaver_etl/beaver_etl/settings.py [hunk 1]
```diff
@@ -17,6 +17,8 @@ class Settings(BaseSettings):
     service_url: str = "http://localhost:8000"
     chunk_size: int = 10
 
+    api_secret_token: str = "73092d292062a7cf2476d56770064c5d88b063d8"
```
Comment (patrykwos) [inline backend/apps/beaver_etl/beaver_etl/settings.py:20]: We store this token in the code for easier development, and it's a temporary solution, right?

Comment (nikitazigman) [reply backend/apps/beaver_etl/beaver_etl/settings.py:20]: yep that is just for the local development it will be overwritten by env var

Comment (nikitazigman) [reply backend/apps/beaver_etl/beaver_etl/settings.py:20]: I think you are right. It makes sense to remove it

Comment (nikitazigman) [reply backend/apps/beaver_etl/beaver_etl/settings.py]: done

**Review by patrykwos — APPROVED** [review]

**Pull request merged**